static/oas/v1/stats/penalty.yml

```diff
@@ -0,0 +1,57 @@
+paths:
```

Comment:
If we're putting this both under penalty in the Open API spec then we need to decide how to name these files, singular or plural. If we want to do singular that's fine, and I'll move the Penalty API Open API spec over. However the URL paths and DB tables are plural. But I'm good either way.

Reply:
I moved penalty.yml into penalties.yml